packages/grid/src/docs/Component.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { ComponentHeader, CssVars, TabContainer } from 'storybook/blocks';
+
+import { Row, Column } from '../';
+import { version } from '../../package.json';
+import Description from './Description.mdx';
+import Props from './Props.mdx';
+import Changelog from '../../CHANGELOG.md';
+
+export const VIEWS = ['primary', 'secondary'];
+export const SIZES = ['small', 'middle', 'large'];
```

Comment:
we use the camelCase format for the local constants the similar as for usual variables